### PR TITLE
Added a stream for BU Computer Science online Juypter-book course mat…

### PR DESCRIPTION
**Pull request by jappavoo**
…erial

The associated Dockerfile can be found here:
https://github.com/jappavoo/bu-cs-book-dev

The public container image is here:
https://hub.docker.com/repository/docker/jappavoo/bu-cs-book-dev

And this is being developed in context of this material:
https://github.com/jappavoo/UndertheCovers

### CODE REVIEW

File: kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  name: bu-cs-jupyter-book
+  labels:
+    component.opendatahub.io/name: jupyterhub
+    opendatahub.io/component: 'true'
+    opendatahub.io/notebook-image: 'true'
+  annotations:
+    opendatahub.io/notebook-image-name:
+      "BU CS Jupyter Book"
+    opendatahub.io/notebook-image-url:
+      "https://github.com/aicoe-aiops/openshift-anomaly-detection"
```
Comment (4n4nd) [inline kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml:14]: can you please change this to your image's source?

File: kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    opendatahub.io/notebook-image-name:
```
Comment (4n4nd) [inline kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml:15]: You already have this annotation on line 11

File: kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      "OpenShift Anomaly Detection Notebook Image"
+    opendatahub.io/notebook-image-desc:
+      "Image with jupyter notebooks for anomaly detection and diagnosis discovery in OCP clusters"
```
Comment (4n4nd) [inline kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml:18]: Update this with the description of your image

**Review by 4n4nd — CHANGES_REQUESTED** [review]

File: kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  name: bu-cs-jupyter-book
+  labels:
+    component.opendatahub.io/name: jupyterhub
+    opendatahub.io/component: 'true'
+    opendatahub.io/notebook-image: 'true'
+  annotations:
+    opendatahub.io/notebook-image-name:
+      "BU CS Jupyter Book"
+    opendatahub.io/notebook-image-url:
+      "https://github.com/aicoe-aiops/bu-cs-jupyter-book-notebook"
```
Comment (4n4nd) [inline kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml:14]: I believe this is your url
'https://github.com/jappavoo/bu-cs-book-dev'

**Review by 4n4nd — CHANGES_REQUESTED** [review]

**Comment by jappavoo** [comment]
Ah my bad… I was template matching not really understanding what this stanza really means ;-)

Thanks for your patience.  

                                            Jonathan
                                                  Appavoo

File: kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml (hunk 3 above)
Comment (jappavoo) [reply kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml]: Thanks I have updated the stanzas

File: kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml (hunk 4 above)
Comment (jappavoo) [reply kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml]: Thanks!  Done

File: kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml (hunk 1 above)
Comment (jappavoo) [reply kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml]: done

File: kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml (hunk 2 above)
Comment (jappavoo) [reply kfdefs/base/jupyterhub/notebook-images/bu-cs-jupyterbook.yaml]: done

**Comment by 4n4nd** [comment]
@jappavoo you will need to add the file name of the file that you are adding to this list https://github.com/operate-first/apps/blob/master/kfdefs/base/jupyterhub/notebook-images/kustomization.yaml

Once you do that and fix the CI errors, this should be good to merge 👍

**Comment by jappavoo** [comment]
@4n4nd Thanks for all your help.  Added the new image config to the kustomization.yaml.  Not sure what you meant by CI errors.

**Comment by 4n4nd** [comment]
[Here](https://github.com/operate-first/apps/blob/master/contributing.md) you can find a section for `pre-commit`. This is one of the tests that our bot @sesheta runs on all PRs.

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**